conformance-tests/src/main.rs

```diff
@@ -22,7 +22,11 @@ fn main() {
         .next()
         .expect("expected second arg to be path to ctr binary")
         .into();
-    conformance_tests::run_tests(move |test| run_test(test, &spin_binary, &ctr_binary)).unwrap();
+    let config = conformance_tests::Config::new("canary");
+    conformance_tests::run_tests(config, move |test| {
+        run_test(test, &spin_binary, &ctr_binary)
+    })
+    .unwrap();
 }
 
 fn run_test(
```

Comment:
would be nice to add the conformance tests to the CI. Could be a separate PR though